.github/workflows/upstream-wpt-changes.yml

```diff
@@ -25,7 +25,11 @@ jobs:
         with:
           path: wpt
           repository: 'web-platform-tests/wpt'
-          token: ${{ secrets.GITHUB_TOKEN }}
+          # The token here must be the token that we will use to push to the
+          # WPT repository and not the token used for GitHub actions, because
+          # the checkout action sets up an extraheader authorization override
```

Comment:
nit: typo in “extraheader”?

Reply:
This git setting is actually called `extraheader`. I can add quotes here to make it clearer though and also link to the GitHub actions issue.

Reply:
yes please!